game.py

```diff
@@ -3,9 +3,12 @@
 
 def main():
     pygame.init()
+
```

Comment:
I had some errors but just fixed it!
the button images need to be called after the screen has loaded. & menu_background variable is missing.
I'd change to following order:
    character_selection = pygame.image.load(os.path.join("images", "character_selection.png"))
    pygame.mixer.pre_init(44100,16,2,4096)
    pygame.display.set_caption("Connect!)")
    screen = pygame.display.set_mode((800,600))
    image = pygame.image.load(os.path.join("images", "background.png"))
    play_button_image = pygame.image.load(os.path.join("images", "basic_button.png")).convert()
    play_button = pygame.draw.rect(screen, (190, 190, 255), [500, 200, 121, 61])
    menu_background = pygame.image.load(os.path.join("images", "background.png"))

Apart from that, clicking the button worked magically!! Great work Hannah!